var/spack/repos/builtin/packages/qhull/package.py

```diff
@@ -36,16 +37,13 @@ class Qhull(Package):
 
     homepage = "http://www.qhull.org"
 
-    version('7.2.0', 'e6270733a826a6a7c32b796e005ec3dc',
+    version('2015.2', 'e6270733a826a6a7c32b796e005ec3dc',
             url="http://www.qhull.org/download/qhull-2015-src-7.2.0.tgz")
 
-    version('1.0', 'd0f978c0d8dfb2e919caefa56ea2953c',
+    version('2012.1', 'd0f978c0d8dfb2e919caefa56ea2953c',
             url="http://www.qhull.org/download/qhull-2012.1-src.tgz")
 
-    # https://github.com/qhull/qhull/pull/5
-    patch('qhull-iterator.patch', when='@1.0')
```

Comment:
The patch that @mathstuf wrote no longer works for either version of qhull in Spack so I removed it. I have a feeling the developers applied the patch to 2012.1 and re-released it without telling anyone. See discussion in #420.